src/pg/.gitignore

```diff
@@ -0,0 +1,6 @@
+regression.diffs
+regression.out
+results/
+crankshaft--dev.sql
+crankshaft--dev--current.sql
+crankshaft--current--dev.sql
```

Comment:
the .pyc are now ignored in the top level .gitignore (may be in src/py as well as in release/python)